app/bank/application/deposit_money/deposit_money_command_handler.py

```diff
@@ -9,5 +10,7 @@ def __init__(self, account_repository:AccountRepository):
 
     def handle(self, command:DepositAmountCommand):
         account_filtered = self.account_repository.get_account_by_iban(account_number=command.account_number)
+        if account_filtered is None:
+            raise AccountNotFoundException(account_number=command.account_number)
         account_filtered.funds_amount = account_filtered.funds_amount + command.deposit_amount
```

Comment:
We need to ensure the sum is performed atomically using update SQL operator or we need to lock the row `using .select_for_update()`